src/app/components/editors/shared/sai-editor/sai-editor.component.integration.spec.ts

```diff
@@ -385,6 +386,91 @@ describe('SaiEditorComponent integration tests', () => {
       );
     });
 
+    it('shows error on wrong linked event', () => {
+
+      const mockRows: SmartScripts[] = [
```

Comment:
in order to test this, you don't need the full `SmartScripts` object. So why don't you just use `Partial<SmartScripts>` and just have an array like:

```
[ { id: 1, link: 1, event_type: 0 }, { id: 2, link: 0, event_type: 61 } ]
```

this will fit in just one line of code